cmd/kubelet/app/server.go

```diff
@@ -121,6 +123,49 @@ is checked every 20 seconds (also configurable with a flag).
 HTTP server: The kubelet can also listen for HTTP and respond to a simple API
 (underspec'd currently) to submit a new manifest.`,
 		Run: func(cmd *cobra.Command, args []string) {
+			// short-circuit on verflag
+			verflag.PrintAndExitIfRequested()
+
+			// TODO(mtaufen): won't need this this once dynamic config is GA
+			// set feature gates so we can check if dynamic config is enabled
+			if err := utilfeature.DefaultFeatureGate.SetFromMap(kubeletServer.KubeletConfiguration.FeatureGates); err != nil {
+				glog.Fatal(err)
```

Comment:
I know this was a move, but consider `cmdutil.CheckErr()` here and below?

Reply:
> I know this was a move, but consider cmdutil.CheckErr() here and below?

That will tie us to `pkg/kubectl`.  I'd rather not end up with that linkage.

Reply:
Ok, maybe we should consider taking that out of e.g. kube-proxy and (if it uses it) kube-scheduler in follow-ups.

Reply:
Probably the base checkErr stuff should move out of kubectl.